sky/dashboard/src/components/infra.jsx

```diff
@@ -654,42 +695,102 @@ export function ContextDetails({ contextName, gpusInContext, nodesInContext }) {
                 <table className="min-w-full text-sm">
                   <thead className="bg-gray-100">
                     <tr>
+                      <th className="p-3 text-left font-medium text-gray-600">
+                        {displayTitle}
```

Comment:
This adds a new column `context` to the table, which is not necessary since we're in the context detail page, WDYT?